etl/moped_projects/README.md

```diff
@@ -18,6 +18,6 @@ docker build . -t atddocker/vz-moped-join
 ```
 
 ```
-docker run -it --env-file env_file atddocker/vz-moped-join /bin/bash
+docker run -it --rm --network host --env-file env_file atddocker/vz-moped-join /bin/bash
```

Comment:
Not a big deal here, but the defaulting to the use of `--network host` defeats the network isolation which is a feature of the docker ecosystem and very generally will lead to port conflicts unless care is taken to make sure none get reused, and this tracking of used ports is maintained as new docker containers are added to the host.

Reply:
the ETL needs to connect to local moped and hasura, so this  is a shortcut to specifying the docker network(s) this container needs to reach.

in this case, i guess the care being taken is the use of the `--rm` flag?

Reply:
The `--rm` flag means to clean up the container when it exits instead of hanging around to be restarted or cleaned up by some a `--remove-orphans` command. That's how I think of it at least .. an 'autoremove' feature.

Reply:
thanks for pushing for the internal host—i made the readme and env template with this change ✨ 

https://github.com/cityofaustin/vision-zero/commit/0194b64d5dbe359c6bfd40a394b0cc1a99d19dbe